mirgecom/diffusion.py

```diff
@@ -47,19 +49,20 @@ def grad_flux(discr, u_tpair, *, quadrature_tag=DISCR_TAG_BASE):
     r"""Compute the numerical flux for $\nabla u$."""
     actx = u_tpair.int.array_context
 
-    dd = u_tpair.dd
-    dd_quad = dd.with_discr_tag(quadrature_tag)
-    dd_allfaces_quad = dd_quad.with_dtag("all_faces")
+    dd_trace = u_tpair.dd
```

Comment:
`u_tpair.dd` already contains a discr tag. Deprecate `quadrature_tag`?

Reply:
This was one of the cases where `dd.discretization_tag` and `quadrature_tag` were different. I moved the quadrature projection up to the top level diffusion operator code and removed the `quadrature_tag` argument here in #744.